### PR TITLE
test(community): Block/Unblock community member

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #7052

### What does the PR do

Add test with check block/unblock functionality for community members

PS: @jrainville I stole one of your methods :D

### CODE REVIEW

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 1]
```diff
@@ -331,3 +338,22 @@ def toggle_pin_message_at_index(self, message_index: int):
     def check_pin_count(self, wanted_pin_count: int):
         pin_text_obj = wait_and_get_obj(CommunityScreenComponents.PIN_TEXT.value)
         verify_equals(pin_text_obj.text, wanted_pin_count)
+
+    def block_member(self, member: str):
+        right_click_obj_by_wildcards_name(CommunityUserListPanel.COMMUNITY_MEMBER_PLACEHOLDER.value, member)
+        block_user_obj = get_obj(CommunityMembersContextMenu.BLOCK_MENU_ITEM.value)
+        hover_obj(block_user_obj)
```
Comment (MishkaRogachev) [inline test/ui-test/src/screens/StatusCommunityScreen.py:345]: like this trick

**Review by MishkaRogachev — APPROVED** [review]
LGTM

File: test/ui-test/src/screens/StatusCommunityScreen.py (hunk 1 above)
Comment (jrainville) [reply test/ui-test/src/screens/StatusCommunityScreen.py:345]: I'm not sure why you need to hover the object before clicking on it?
Possible to block and unblock users

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py [hunk 2]
```diff
@@ -122,3 +126,20 @@ def step(context, message_index):
 def step(context, amount):
     _statusCommunityScreen.check_pin_count(amount)
 
+@Then("the user blocks member with name |any|")
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:129]:
```suggestion
@Then("the user blocks the member named |any|")
```

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py [hunk 3]
```diff
@@ -122,3 +126,20 @@ def step(context, message_index):
...
+def step(context, member):
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:130]: Same thing for the others. It makes it clearer that we refer to the name
```suggestion
def step(context, member_name):
```

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py [hunk 4]
```diff
@@ -122,3 +126,20 @@ def step(context, message_index):
...
+    _statusCommunityScreen.block_member(member)
+
+@Then("the user can see that |any| was blocked")
+def step(context, member):
+    _statusCommunityScreen.check_member_is_blocked(member, True)
+
+@Then("the user unblocks member with name |any|")
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:137]:
```suggestion
@Then("the user unblocks the member named |any|")
```

**Review by jrainville — COMMENTED** [review]
Was the issue https://github.com/status-im/status-desktop/issues/7052 already fixed?
I don't see any QML or Nim code that would fix the issue. 
I'm guessing it was fixed in master and this only adds a test

File: test/ui-test/testSuites/suite_status/tst_communityMembersFlow/test.feature [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Feature: Status Desktop Community Members
+
+	As a user I want to use community chat functionality.
+
+  Background:
+
+       Given the user starts the application with a specific data folder ../../../fixtures/community_members
+
+    @mayfail
+    Scenario: As an admin of comminty i want to be able block and unblock members
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/tst_communityMembersFlow/test.feature:10]:
```suggestion
    Scenario: As an admin of a community, I want to be able block and unblock members
```

File: test/ui-test/src/screens/StatusCommunityScreen.py (hunk 1 above)
Comment (borismelnik) [reply test/ui-test/src/screens/StatusCommunityScreen.py:345]: as @MishkaRogachev and @endulab mentioned - sometimes it doesn't work. :)

**Comment by borismelnik** [comment]
> Was the issue #7052 already fixed?
> I don't see any QML or Nim code that would fix the issue.
> I'm guessing it was fixed in master and this only adds a test

yes I can't reproduce the bug

**Comment by jrainville** [comment]
@borismelnik can you rebase please, I'll review after

**Pull request closed without merging**